server/src/main/java/org/elasticsearch/health/HealthPeriodicLogger.java

```diff
@@ -212,6 +212,18 @@ static Map<String, Object> convertToLoggedFields(List<HealthIndicatorResult> ind
             );
         });
 
+        // message field. Show the non-green indicators if they exist.
+        List<String> nonGreen = indicatorResults.stream()
+            .filter(p -> p.status().xContentValue().equals("green") == false)
+            .flatMap(p -> Stream.of(p.name()))
```

Comment:
This should be able to be something like `.map(HealthIndicatorResult::name)` right? I don't think we need to map the stream of names?